Gemfile.lock

```diff
@@ -441,7 +442,7 @@ GEM
       crack (>= 0.3.2)
       hashdiff (>= 0.4.0, < 2.0.0)
     webrick (1.8.1)
-    win32-api (1.10.1-universal-mingw32)
```

Comment:
We need to for correct gem installation on windows

Reply:
bundling with 3.1.6 resolved the problem, looks related to https://github.com/rubygems/rubygems/issues/5743